app/controllers/api/verify/password_confirm_controller.rb

```diff
@@ -97,26 +97,7 @@ def save_in_person_enrollment(user, profile)
           profile: profile,
           user: user,
           current_address_matches_id: user_session.dig(:idv, :applicant, :same_address_as_id),
-          selected_location_details: {
-            'name' => 'BALTIMORE — Post Office™',
-            'streetAddress' => '900 E FAYETTE ST RM 118',
-            'city' => 'BALTIMORE',
-            'state' => 'MD',
-            'zip5' => '21233',
-            'zip4' => '9715',
-            'phone' => '555-123-6409',
-            'hours' => [
-              {
-                'weekdayHours' => '8:30 AM - 4:30 PM',
-              },
-              {
-                'saturdayHours' => '9:00 AM - 12:00 PM',
-              },
-              {
-                'sundayHours' => 'Closed',
-              },
-            ],
-          },
+          selected_location_details: user_session.dig(:idv, :applicant, :selected_location_details),
```

Comment:
This is not currently working; I'm writing the location to the session in `UspsLocationsController` but it doesn't persist.

Reply:
fixed in 5c446b790ea610de1fbed5fa909a70925e8cc1e0